docs/user-guide/images.md

```diff
@@ -69,6 +69,20 @@ images in the ECR registry.
 The kubelet will fetch and periodically refresh ECR credentials.  It needs the
 `ecr:GetAuthorizationToken` permission to do this.
 
+Requirements:
+
+- You must be using kubelet version `v1.2.0` or newer.  (e.g. run `/usr/bin/kubelet --version=true`).
```

Comment:
They might also be on CoreOS, which comes with an ancient kubelet in /usr/bin (I put my own binary elsewhere). In that case they would have to double check in systemd, upstart or initd configuration files which binary is actually spawned. I have a PR to just log the version at startup.